docs/modules/ROOT/pages/documentation/payara-server/upgrade-server.adoc

```diff
@@ -3,15 +3,17 @@
 
 Payara Server can be upgraded to new versions from version *5.24.0*
 
+NOTE: This feature is in technical preview.
```

Comment:
Are we going to use the term 'technical preview'?

Reply:
I thought we were going with MVP - minimally viable product (MVP) that can be used right away - but will be further developed and improved upon to offer a richer feature set in future Enterprise releases.